### PR TITLE
validation puzzle

### PR DESCRIPTION
**Pull request by runk**

### CODE REVIEW

File: 07-validation/test/index.coffee [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        colors:
+          header: '#ff6'
+          footer: '#ccccff'
+      browsers: [
+        {name: 'chrome', version: 25}
+        {name: 'safari', version: 11}
+      ]
+
+  # !!!!!
+  # Add more tests for different data users can ship to our app!
```
Comment (jamesramsay) [inline 07-validation/test/index.coffee]: data users?

Comment (runk) [inline 07-validation/test/index.coffee]: Erm. Yeah, a bit confusing. 

How about "... for different data that users can ship ... "?

**Comment by jamesramsay** [comment]
Reviewing the problem from a higher level, if a candidate is not familiar with Node, using npm to find and install al library might be a bit too much. Also, the expectation to read documentation for probably a completely unfamiliar library and implement against it is probably also pretty tough.

**Comment by jamesramsay** [comment]
Good start. Will propose some changes in a separate PR.

**Pull request merged**